runs/2017/pyladies-praha-podzim-cznic/info.yml

```diff
@@ -0,0 +1,134 @@
+title: Začátečnický kurz Pyladies
+subtitle: Praha (CZ.NIC) - podzim 2017
+time: 18:00–20:00
+place: budova CZ.NIC, Milešovská 5, Praha 3
+description: Naučte se Python vážně od začátku. Žádné předchozí znalosti nejsou potřeba.
+long_description: |
+    Tady najdeš všechny materiály k brněnské verzi začátečnického kurzu
+    PyLadies.
+
+    Stránky samotných PyLadies jsou na [http://pyladies.cz][Pyladies].
+
+    Jednotlivé lekce jsou určeny naprostým začátečnicím.
+    Instrukce jsou uvedeny pro operační systémy Linux, Windows i macOS.
+
+    [PyLadies]: http://pyladies.cz/
+derives: pyladies
+vars:
+  user-gender: f
+  pyladies: true
+  coach-present: true
+plan:
+- base: install
+  date: 2017-09-19
+  materials:
+  - title: Úvodní prezentace
+    url: http://pyladies.cz/v1/s001-install/uvod-praha.html
+  - +merge
+  - title: Instrukce a domácí projekty (PDF)
+    url: https://github.com/PyLadiesCZ/pyladies.cz/raw/master/original/v1/s001-install/handout/handout.pdf
+    type: homework
+- base: hello
+  slug: hello-world
+  date: 2017-09-26
+  materials:
+  - lesson: beginners/hello-world
+    type: lesson
+  - lesson: beginners/print
+    type: lesson
+  - lesson: beginners/variables
+    type: lesson
+  - lesson: beginners/comparisons
+    type: lesson
+  - title: Domácí projekty (PDF)
+    url: https://github.com/PyLadiesCZ/pyladies.cz/raw/master/original/v1/s002-hello-world/handout/handout2.pdf
+    type: homework
+  - title: "*Nebo* anebo *a* (bonus)"
+    lesson: beginners/and-or
+    vars:
+      bonus: true
+- base: loops
+  date: 2017-10-03
+  materials:
+  - +merge
+  - title: Domácí projekty (PDF)
+    url: https://github.com/PyLadiesCZ/pyladies.cz/raw/master/original/v1/s003-looping/handout/handout3a.pdf
+    type: homework
+- title: Spolupráce a Opensource - Git
+  slug: vcs
+  date: 2017-10-10
+  materials:
+  - lesson: git/git-collaboration-2in1
+    type: lesson
+  - title: Gitový tahák
+    url: https://github.com/pyvec/cheatsheets/raw/master/basic-git/basic-git-cs.pdf
+    type: cheatsheet
+  - title: Domácí projekty (PDF)
+    url: https://github.com/PyLadiesCZ/pyladies.cz/raw/master/original/v1/s003-looping/handout/handout3b.pdf
+    type: homework
+- base: def-str
+  date: 2017-10-17
+  materials:
+  - +merge
+  - title: Domácí projekty (PDF)
+    url: http://pyladies.cz/v1/s004-strings/handout/handout4.pdf
+    type: homework
+- title: Procvičování
+  date: 2017-10-24
+  slug: exercises
+  materials: []
+- base: tests
+  slug: testing
```

Comment:
This overrides the slug form the base (`test` -> `testing`). I'd rather get rid of this, but I kept it, so the links are consistent with spring run.